src/SFA.DAS.EmployerIncentives.Commands/Services/IncentivePaymentProfilesService.cs

```diff
@@ -1,39 +1,37 @@
 ﻿using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
-using Microsoft.Extensions.Options;
 using SFA.DAS.EmployerIncentives.Domain.Interfaces;
-using SFA.DAS.EmployerIncentives.Infrastructure.Configuration;
+using SFA.DAS.EmployerIncentives.Domain.ValueObjects;
+using SFA.DAS.EmployerIncentives.Enums;
 
 namespace SFA.DAS.EmployerIncentives.Commands.Services
 {
     public class IncentivePaymentProfilesService : IIncentivePaymentProfilesService
     {
-        private readonly ApplicationSettings _applicationSettings;
-
-        public IncentivePaymentProfilesService(IOptions<ApplicationSettings> applicationSettings)
-        {
-            _applicationSettings = applicationSettings.Value;
-        }
-
-        public Task<IEnumerable<Domain.ValueObjects.IncentivePaymentProfile>> Get()
+        public Task<IEnumerable<IncentivePaymentProfile>> Get()
```

Comment:
not much point it being async if we don't utilise it. i know it was that before but we might as well tidy it up now.